### DISCUSSION TITLE
Do we have a scheduler like `ReduceLROnPlateau`?

### DISCUSSION THREAD

**Discussion by nalzok** [google-deepmind/optax#422] [Q&A]
PyTorch users have [`torch.optim.lr_scheduler.ReduceLROnPlateau`](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html#reducelronplateau), which can monitor the loss curve and dynamically reduce the learning rate when it seems to plateau. Do we have something similar in Optax?
